playbooks/10 - SP - GDPR Framework/Get Updated Data Breach Report from DPO.json

```diff
@@ -581,14 +581,14 @@
                     "mitigationActions": "{{vars.steps.Update_DPO_Report.input['whatMitigationActionsHaveBeenImplemented?']}}",
                     "noOfAffectedUsers": "{{vars.steps.Update_DPO_Report.input['no.OfAffectedUsers']}}",
                     "whereDataWasHosted": "{{vars.steps.Update_DPO_Report.input['whereDataWasHosted?']}}",
-                    "wasTheDataEncrypted": "{{\"ChooseYesOrNo\" | picklist(vars.steps.Update_DPO_Report.input['wasTheDataEncrypted?'], \"@id\")}}",
+                    "wasTheDataEncrypted": "{{\"YesNoInput\" | picklist(vars.steps.Update_DPO_Report.input['wasTheDataEncrypted?'], \"@id\")}}",
                     "whatWasTheDataFormat": "{{vars.steps.Update_DPO_Report.input['whatWasTheDataFormat?']}}",
                     "whatWasTheDataSource": "{{vars.steps.Update_DPO_Report.input['whatWasTheDataSource?']}}",
                     "repercussionsOfBreach": "{{vars.steps.Update_DPO_Report.input['whatAreTheRepercussionsOfBreach?']}}",
-                    "wasTheExposureResolved": "{{\"ChooseYesOrNo\" | picklist(vars.steps.Update_DPO_Report.input['wasTheExposureResolved?'], \"@id\")}}",
+                    "wasTheExposureResolved": "{{\"YesNoInput\" | picklist(vars.steps.Update_DPO_Report.input['wasTheExposureResolved?'], \"@id\")}}",
                     "noOfAffectedDataRecords": "{{vars.steps.Update_DPO_Report.input['no.OfAffectedDataRecords']}}",
                     "anyAdditionalInformation": "{{vars.steps.Update_DPO_Report.input.anyAdditionalInformation}}",
-                    "isHarmRiskOrMisuseForeseeable": "{{\"ChooseYesOrNo\" | picklist(vars.steps.Update_DPO_Report.input['riskOrMisuseForeseeable'], \"@id\")}}"
+                    "isHarmRiskOrMisuseForeseeable": "{{\"YesNoInput\" | picklist(vars.steps.Update_DPO_Report.input['riskOrMisuseForeseeable'], \"@id\")}}"
```

Comment:
I didn't get this name/field. Can you please explain?

Reply:
The picklist was previously named "ChooseYesOrNo" which is now renamed "YesNoInput".

Reply:
I mean to say Field name